### DISCUSSION TITLE
CVE-2023-48795 (crypto/ssh)

### DISCUSSION THREAD

**Discussion by helmut72** [drakkan/sftpgo#1490] [General]
Please have a look:
https://groups.google.com/g/golang-announce/c/-n5WqVC18LQ

Maybe an update is needed.

**Comment by drakkan** [comment]
As the maintainer of x/crypto/ssh I am aware of this issue and have been working with the Go team to fix it.

People/companies who care about issues like this and that SFTPGo is maintained in general perhaps should stop being leeches (in IT terms) otherwise sooner or later they will find themselves using a project that is no longer maintained and problems like this and other bugs will no longer be fixed. Thank you

**Comment by helmut72** [comment]
I woke up in the night and sometimes I read r/selfhosted or other sites for some minutes. Then I found this:
https://www.reddit.com/r/selfhosted/comments/18ko7jw/headsup_golang_ssh_cve202348795/

... and sent it only as a friendly information.
